test/prefer-native-test.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const AddonFactory = require('../');
+const expect = require('chai').expect;
+const helpers = require('broccoli-test-helper');
+const vm = require('vm');
+const fs = require('fs');
+const path = require('path');
+const templatePath = path.resolve(__dirname + '/assets/browser-fetch.js.t');
+const RSVP = require('rsvp');
+
+const testCode = `
+define('fetch-test', ['fetch'], function(_fetch) {
+  _fetch.default();
+});
+require('fetch-test');
+`;
+
+[true, false].forEach(flag => {
+  describe(`Build browser assets with preferNative = ${flag}`, function() {
+    let output, subject, addon;
+
+    beforeEach(async function() {
```

Comment:
we can drop `async` here, as it doesn't appear we have an await in the function

Reply:
good catch